### DISCUSSION TITLE
TuringCourses/political/marxism_basic/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#168] [General]
# TuringCourses/political/marxism_basic/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/political/marxism_basic/

**Comment by 626lt** [comment]
吴旭平老师说以后一半开马原（H），一半开普通的课，所以选课的压力会相对增大。如果你有幸选上吴老师的课但又不想要花时间，以下的混分学私货适合你（x）：平时的课大部分不听也没关系，但要保证一定的发言次数（2-3)次，最好是一开始上课的时候最容易发言，那个时候一般是话题引入，积极举手很容易就混到发言次数了。期末复习的时候可以买一本马原的题库来刷一下，注意有一些知识性的问题，比如以下哪一著作是马克思和恩格斯合著的等等需要看一下课本来记忆一下。大题要把原理背下来，可以跟着b站上的视频进行一个突击的背诵，最好有针对性地结合一下时事来背诵。最后你可以相信吴老师的给分，在查老师上的均绩可见一斑。

**Comment by WangHanZJU** [comment]
注意：吴旭平老师在2024年后评分标准有所改变，具体请参考他第一节课的ppt。
